packages/frontend/apps/electron/src/preload/bootstrap.ts

```diff
@@ -2,12 +2,11 @@ import '@sentry/electron/preload';
 
 import { contextBridge } from 'electron';
 
-import { appInfo, getElectronAPIs } from './electron-api';
+import { apis, appInfo, events, requestWebWorkerPort } from './electron-api';
 import { sharedStorage } from './shared-storage';
 
-const { apis, events } = getElectronAPIs();
-
 contextBridge.exposeInMainWorld('__appInfo', appInfo);
 contextBridge.exposeInMainWorld('__apis', apis);
 contextBridge.exposeInMainWorld('__events', events);
 contextBridge.exposeInMainWorld('__sharedStorage', sharedStorage);
+contextBridge.exposeInMainWorld('__requestWebWorkerPort', requestWebWorkerPort);
```

Comment:
the `requestWebWorkerPort` is a function and it returns obj with function, is it work for bridge?

Reply:
Yes, this is supported. https://www.electronjs.org/docs/latest/api/context-bridge#:~:text=Function%20values%20are%20proxied%20to%20the%20other%20context%20and%20all%20other%20values%20are%20copied%20and%20frozen.%20Any%20data%20/%20primitives%20sent%20in%20the%20API%20become%20immutable%20and%20updates%20on%20either%20side%20of%20the%20bridge%20do%20not%20result%20in%20an%20update%20on%20the%20other%20side.